### ISSUE TITLE
FR: pgproxy release artifacts / OCI images

### ISSUE THREAD

**Issue by philpennock** [tailscale/tailscale#11351]
### What are you trying to do?

I would like a project by our developers to just use pgproxy, to simplify a lot of the design and cut out much work.

The dev lead in question writes:
>  I don't see any officially supported or released artifacts for it.  Ideally we want to run something officially supported by tailscale off of dockerhub.

### How should we solve this?

Provide `tailscale/pgproxy` images on https://hub.docker.com/u/tailscale

**Comment by rafitox** [comment]
I agree! It would be really cool. I implemented it myself (experimental, i'd say) but it works. it would be nice to have an official image. You may check my implementation if you want: https://github.com/rafitox/pgproxy_tailscale i used chainguard golang base image and built on top of it

**Comment by casperfrx** [comment]
Oh cool, looks like I wasn't the only one looking for this. I was confused how to invoke the cmd commands like that
